api/v1/ptpconfig_webhook.go

```diff
@@ -90,8 +90,16 @@ func (output *ptp4lConf) populatePtp4lConf(config *string, ptp4lopts *string) er
 func (r *PtpConfig) validate() error {
 	profiles := r.Spec.Profile
 	for _, profile := range profiles {
+		if profile.Ptp4lConf == nil {
```

Comment:
if you leave it empty it will crash here

https://github.com/openshift/ptp-operator/pull/160/files#diff-49d9e80697fdeec0d139aa476343d0fbb1a04127dc13bb69654138b33021f6d4R118